.github/workflows/dep-diff-pull_request.yml

```diff
@@ -5,7 +5,9 @@ name: Dependency Tree Input Builder
 on:
   pull_request:
     branches:
-    - main
+      - main
+    paths:
+      - "**/pom.xml"
```

Comment:
@luck3y is the pull-player kickoff of the ci-wf-org jobs still gated by completion of this job? If so we should rethink that.

Reply:
@bstansberry As far as I am aware there is no gating, they run async of one another.